cloudbuild.yaml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: 'gcr.io/cloud-builders/git'
```

Comment:
Is this correct: 
Each client library will copy the first 6 lines of this file into its own.

There's probably some value in standardizing the way this is done to allow re-use of a yaml fragment, but if it's this tiny, probably not very much value.

Reply:
I believe the way to standardize is to essentially bundle these into it's own container so that it runs in one step.